string planning_group
geometry_msgs/Pose pose
---
string[] joint_name
float64[] position